config.fullstack.test.yaml

```diff
@@ -7,8 +7,9 @@ default_project: &default_project
               - path: projects/v1/default.wmf.yaml
                 options: &default_options
                   parsoid:
-                    host: http://parsoid-external-ci-access.beta.wmflabs.org/w/rest.php
-                    grace_ttl: 1000000
+                    host: https://parsoid-external-ci-access.beta.wmflabs.org/w/rest.php
+                    # disabled_storage: true
+                    grace_ttl: 1000007
```

Comment:
These changes should probably not be in this patch.
Though I can't test without https...